README.md

```diff
@@ -269,6 +270,26 @@ spectator.query(byLabel('By label'));
 spectator.query(byText('By text'));
 ```
 
+
+#### Mocking components
+
+If you need to mock components, you can use the [ng-mocks](https://github.com/ike18t/ng-mocks) library. Instead of using `CUSTOM_ELEMENTS_SCHEMA` which might hide some issues and won't help you to set inputs, outputs, etc, `ng-mocks` will auto mock the inputs, outputs, etc for you.
```

Comment:
redudant space

Reply:
Not sure what you mean by that but I've removed the empty lines around the title